mdx-models/src/main/java/com/mx/path/model/mdx/accessor/payment/PaymentBaseAccessor.java

```diff
@@ -51,6 +53,52 @@ public PaymentBaseAccessor(AccessorConfiguration configuration) {
     super(configuration);
   }
 
+  /**
+   * Enrollment status
+   *
+   * @return
+   */
+  @GatewayAPI
+  @API(description = "Get the Enrollment status")
+  public AccessorResponse<Enrollment> enrollmentStatus() {
```

Comment:
[Suggestion] Remove the word "status", since we are getting an Enrollment object.

Reply:
updated